ggml/CMakeLists.txt

```diff
@@ -185,6 +185,9 @@ option(GGML_OPENCL_PROFILING                "ggml: use OpenCL profiling (increas
 option(GGML_OPENCL_EMBED_KERNELS            "ggml: embed kernels"                             ON)
 option(GGML_OPENCL_USE_ADRENO_KERNELS       "ggml: use optimized kernels for Adreno"          ON)
 
+# toolchain for vulkan-shaders-gen
+set   (GGML_SHADERS_GEN_TOOLCHAIN "" CACHE FILEPATH "ggml: toolchain file for vulkan-shaders-gen")
```

Comment:
This option should have a `GGML_VULKAN` prefix. For example, it can be named `GGML_VULKAN_SHADERS_GEN_TOOLCHAIN`

Reply:
Do you have more comments or is that it? The PR has been open for way too long now and I'd like to get it merged.

Reply:
No, merge at will.

Reply:
Addressed. Thanks.